src/docs/howto-listing/time-slots/index.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Modify booking time intervals in FTW-hourly
+slug: modify-time-intervals
+updated: 2022-08-29
+category: how-to-listing
+ingress:
+  This guide describes how to modify booking time intervals in
+  FTW-hourly
+published: true
+---
+
+In Flex, listings can have either day-based or time-based availability.
+For listings with time-based availability, the available
+[time slots](https://www.sharetribe.com/api-reference/marketplace.html#time-slots)
+are returned from the API as continuous stretches of time. The client
+application must therefore split the availability into suitable booking
+lengths.
+
+The default behavior of the FTW-hourly template is to split the
+continuous availability stretch into one hour bookable intervals. This
+how-to guide illustrates a handful of different cases on modifying
+booking lengths.
+
+## Set booking length to 30 minutes
+
+The simplest use case is to create uniform 30 minute booking slots.
+Start with adding a constant for the booking length in minutes.
+
+```js
+const timeSlotMinutes = 30;
+```
+
+The time slot handling is done using a few helper functions in
+src/util/dates.js
+
+```shell
+└── src
+    └── util
+        ├── dates.js
+```
+
+- _getStartHours_ and _getEndHours_ use a helper function
+  _getSharpHours_
+- _getSharpHours_, in turn, uses _findBookingUnitBoundaries_, which is a
+  recursive function that takes a _nextBoundaryFn_ parameter that it
+  uses to determine the correct booking time boundaries
+- the function passed to _findBookingUnitBoundaries_ as _nextBoundaryFn_
+  by default is _findNextBoundary_, which is what we need to modify
+  first.
```

Comment:
Feel free to argue against this, but what would you think about having a more detailed explanations of what these functions do?

Reply:
I wrote some more explanation for the functions named in these bullet points